iottalkpy/dan.py

```diff
@@ -439,6 +439,13 @@ def register(self, url, on_signal, on_data,
             json.dumps({'state': 'offline', 'rev': rev}),
             retain=True,
         )
+
+        # Enable TLS support if the received scheme is mqtts.
+        #
+        # Ref: https://github.com/eclipse/paho.mqtt.python#tls_set
+        if str(metadata.get('url', {}).get('scheme', '')).lower() == 'mqtts':
```

Comment:
這些欄位根據 spec 保證有，如果沒有程式必須 crash。
```suggestion
        if metadata['url']['scheme'].lower() == 'mqtts':
```

Reply:
已修改